### DISCUSSION TITLE
Are there steps to enable on existing postgres?

### DISCUSSION THREAD

**Discussion by recurrence** [hydradatabase/hydra#129] [General]
EG: Can I just make and make install it since it's an extension?

I didn't see an install in the Makefile but maybe I missed it.

**Comment by wuputah** [comment]
Yeah you can certainly do a native build of the columnar and add it to your local Postgres. The Hydra image / service has more extensions than just the columnar extension but obviously columnar is the most fun part.

Pre-reqs: 
* You'll need a standard build env and dev postgres install... you can grab the packages from the [Dockerfile](https://github.com/hydradatabase/hydra/blob/main/columnar/Dockerfile#L7)
* You'll need liblz4-dev and libzstd-dev for a full build, or you can pass `--without-...` as applicable to `./configure`

(It will also build natively on Mac, but I don't have a list of homebrew packages for ya.)

After that, this should work:

```        
cd ./columnar
./configure
cd src/backend/columnar
sudo make install
```

**Reply by recurrence** [reply]
Wicked, thanks!
